e2e-tests/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'java'
+    id 'checkstyle'
+    alias(libs.plugins.allure)
+}
+
+ext {
+    kafkaVersion = "3.8.0"
+    aspectjVersion = "1.9.21"
+    allureVersion = "2.27.0"
+}
+
+dependencies {
+    implementation project(":contract")
+    implementation "org.apache.kafka:kafka_2.13:${kafkaVersion}"
+    implementation "org.aspectj:aspectjweaver:${aspectjVersion}"
+    implementation "commons-io:commons-io:2.16.1"
+    implementation "org.testng:testng:7.10.0"
+    implementation "com.codeborne:selenide:7.2.3"
```

Comment:
Let's please add this to the catalog as well

Reply:
This is e2e we'll make it more "gradle-style" later, current task is to migrate